notebooks/01_data_on_manifolds.ipynb

```diff
@@ -0,0 +1,669 @@
+{
```

Comment:
<p>ditto, try <code>ax.set_title('Cities on the earth.');</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/786/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
Done.